test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/DefaultTagHelperContentTest.cs

```diff
@@ -562,17 +544,14 @@ public void Fluent_SetContent_AppendFormat_WritesExpectedContent()
         {
             // Arrange
             var tagHelperContent = new DefaultTagHelperContent();
-            var expected = new[] { "First ", "Second Third" };
+            var expected = "First Second Third";
             var i = 0;
 
             // Act
             tagHelperContent.SetContent("First ").AppendFormat("{0} Third", "Second");
 
             // Assert
-            foreach (var value in tagHelperContent)
-            {
-                Assert.Equal(expected[i++], value);
-            }
+            Assert.Equal(expected, tagHelperContent.GetContent());
```

Comment:
Same reasoning as above. Since this is an implementation detail of `BufferedHtmlContent` I am not verifying it here. `BufferedHtmlContent` has its own tests to verify that it is stored as a list of entries.